modules/Data_Visualization/Data_Visualization.Rmd

```diff
@@ -931,7 +930,7 @@ rp_fac_plot
 ## plotly
 
 ```{r}
-#install.packages("plotly")
+#install.packages(plotly)
```

Comment:
```suggestion
#install.packages("plotly")
```
Was this an automatic formatting? very strange

Reply:
Ah I see what happened. We wanted to remove them around the `library()` function